### PR TITLE
Added classifier_face_emotion_openvino

### PR DESCRIPTION
**Pull request by rhedgeco**

### CODE REVIEW

File: capsules/classifier_face_emotion_openvino/config.py [hunk 1]
```diff
@@ -0,0 +1 @@
+emotion_types = ['neutral', 'happy', 'sad', 'surprise', 'anger']
```
Comment (apockill) [inline capsules/classifier_face_emotion_openvino/config.py]: Tbh having a config for one line probably isn't necessary :P You can have it in the backend and import it from capsule.py to do the same thing

File: capsules/classifier_face_emotion_openvino/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# About the Capsule
+## Usage
+This capsule is for detecting the the emotion of faces in video. It should be
```
Comment (BryceBeagle) [inline capsules/classifier_face_emotion_openvino/README.md]: the the

File: capsules/classifier_face_emotion_openvino/meta.conf [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[about]
+api_compatibility_version = 0.2
```
Comment (BryceBeagle) [inline capsules/classifier_face_emotion_openvino/meta.conf]: Nitpick: new line at end of file

**Review by BryceBeagle — CHANGES_REQUESTED** [review]

File: capsules/classifier_face_emotion_openvino/meta.conf (hunk 3 above)
Comment (BryceBeagle) [reply capsules/classifier_face_emotion_openvino/meta.conf]: For reference: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Review by BryceBeagle — APPROVED** [review]

**Review by velovix — APPROVED** [review]

**Review by apockill — APPROVED** [review]

**Comment by apockill** [comment]
This is ready for merging. @rhedgeco feel free to merge at your leisure, unless you anticipate further changes.

**Comment by rhedgeco** [comment]
lol gimme that shirt

**Pull request merged**